projects/vanilla/src/app/global.component.ts

```diff
@@ -115,4 +151,19 @@ export class GlobalComponent implements OnChanges {
       document.documentElement.style.setProperty(`--${name}-${index * 100 + 300}`, value);
     }
   }
+
+  unsetColorVariants(name: ColorVariants) {
+    // variants from 100-200
```

Comment:
Here, as you unset all color variants, you can use only one for loop instruction from 1 to 6.

```javascript
for (let index = 1; index < 6; index++) {
  document.documentElement.style.removeProperty(`--${name}-${index * 100}`);
}
```

Reply:
Bien vu